pom.xml

```diff
@@ -3,9 +3,9 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>com.example</groupId>
+	<groupId>edu.umich.its</groupId>
 	<artifactId>spanish-placement-exams</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
+	<version>0.1.0</version>
 
```

Comment:
I see that `unirest` library is used for http call, is there any reason why that is preferred over the usual apache http client library?